### PR TITLE
GameStatus Nested Paragraph console error fix

### PR DESCRIPTION
**Pull request by rosalie0**
- deleted the GuessesRemaining component because it was only conditionally rendering a line of text - this is now conditionally rendered in the component it was called in.
- has a colon in between the note and guesses remaining instead of unicode no-break character
- doing this fixes the nested paragraph bug

### CODE REVIEW

File: src/components/RoomView/GameStatus.tsx [hunk 1]
```diff
@@ -22,8 +22,12 @@ const GameStatus = () => {
   const operativeGuessingNote: string = 'guess a word or click end turn';
 
   useEffect(() => {
-    const isPlayerTeamOneOperative: boolean = teamOneOperatives.map((operative) => operative.playerId).includes(playerId);
-    const isPlayerTeamTwoOperative: boolean = teamTwoOperatives.map((operative) => operative.playerId).includes(playerId);
+    const isPlayerTeamOneOperative: boolean = teamOneOperatives
```
Comment (rosalie0) [inline src/components/RoomView/GameStatus.tsx:25]: this is prettier, i didn't change anything here.

File: src/components/RoomView/GameStatus.tsx [hunk 2]
```diff
@@ -91,18 +95,14 @@ const GameStatus = () => {
     }
   }, [gameStatus]);
 
-  return (
```
Comment (rosalie0) [inline src/components/RoomView/GameStatus.tsx:94]: changed the wording for what we return

File: src/components/RoomView/GameStatus.tsx [hunk 3]
```diff
@@ -91,18 +95,14 @@ const GameStatus = () => {
...
-            {playerNote}
-            {'\u00A0'}
-            <GuessesRemaining />
-          </p>
-        )}
-        {gameStatus === 'ready' && <p className='gameStatus'>Waiting to begin the game!</p>}
-    </>
-  );
+  if (gameStatus === 'ready') return <p className="gameStatus">Waiting to begin the game!</p>;
+  else
```
Comment (ocjarman) [inline src/components/RoomView/GameStatus.tsx:99]: i like this change a lot!

**Review by ocjarman — APPROVED** [review]
looks good!

**Pull request merged**